KarhooSDKTests/TestCases/Service/Trips/BookingInteractor/KarhooBookingInteractorSpec.swift

```diff
@@ -120,7 +116,6 @@ class KarhooBookingInteractorSpec: XCTestCase {
      */
     func testRequestFails() {
         let tripBooking = TripBooking(quoteId: "some",
-                                      passengers: passengers,
```

Comment:
Yep, This file too, add all the Passengers back

Reply:
done